include/ensmallen_bits/problems/dtlz1.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file dtlz1.hpp
+ * @author Rahul Ganesh Prabhu
+ *
+ * Definition of the first DTLZ function.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_PROBLEMS_DTLZ_ONE_HPP
+#define ENSMALLEN_PROBLEMS_DTLZ_ONE_HPP
+
+namespace ens {
+namespace test {
+
+class DTLZ1
+{
+ public:
+ 	//! Initialize the DTLZ1 function.
```

Comment:
The Indentation is off, but I think you already mentioned this in IRC.

Reply:
Have you folks considered using [`.clang-format`](https://clang.llvm.org/docs/ClangFormat.html) for autoformatting and adding an [`.editorconfig`](https://editorconfig.org/) to enforce the project style proactively?

Reply:
For mlpack we did setup a Jenkins Job that checks the style, I think we should do the same for ensmallen. About autoformatting, haven't tested it yet, maybe this is an option saves us some time.